app/views/component/detail/_address.html.haml

```diff
@@ -2,7 +2,8 @@
   %span{ itemscope: '', itemtype: 'http://schema.org/PostalAddress', itemprop: 'address' }
     %section.icon-text-block
       %i.fa.fa-map-marker
-      %h4.annotated-text-block Location
+      %h2.visuallyhidden= "Address"
+      %h3.annotated-text-block Location
```

Comment:
This is correctly mentioned by the screen reader

Reply:
Would you like me to revert these changes? This is about the page skipping header structure and not about the screen reader per say.